reader.go

```diff
@@ -1578,3 +1578,14 @@ func extractTopics(members []GroupMember) []string {
 
 	return topics
 }
+
+func describeOffset(offset int64) string {
```

Comment:
An alternative model could be to use a type implementing `fmt.Formatter`?

```go
type humanOffset int64

func (offset humanOffset) Format(w fmt.State, _ rune) {
  ...
}
```
One advantage is it removes the need to allocate an intermediary string to encode the offset into the decimal representation, it can be written directly to the output with `fmt.Fprintf(w, "%d", int64(offset))`.

I don't think it would have a measurable impact tho, up to you to choose with model you think would be the most expressive.

Reply:
I like it, thanks!